android/build.gradle.kts

```diff
@@ -1,72 +1,64 @@
-import java.io.File
-import java.io.FileInputStream
-import java.util.*
-
 plugins {
     id("com.android.application")
     kotlin("android")
     id("com.google.gms.google-services")
 }
 
+val keystoreProperties = rootDir.loadGradleProperties("signing.properties")
+
 android {
 
     signingConfigs {
-        val keystorePropertiesFile = rootProject.file("signing.properties")
-        if (keystorePropertiesFile.exists()) {
-            create("release") {
-                val keystoreProperties = Properties()
-                keystoreProperties.load(FileInputStream(keystorePropertiesFile))
-                storeFile = rootProject.file("config/release.keystore")
-                storePassword = keystoreProperties.getProperty("ANDROID_SIGNING_KEYSTORE_PASSWORD")
-                keyAlias = keystoreProperties.getProperty("ANDROID_SIGNING_KEY_ALIAS")
-                keyPassword = keystoreProperties.getProperty("ANDROID_SIGNING_KEY_PASSWORD")
-            }
+        create(BuildType.RELEASE) {
+            storeFile = rootProject.file("config/release.keystore")
+            storePassword = keystoreProperties.getProperty("ANDROID_SIGNING_KEYSTORE_PASSWORD")
+            keyAlias = keystoreProperties.getProperty("ANDROID_SIGNING_KEY_ALIAS")
+            keyPassword = keystoreProperties.getProperty("ANDROID_SIGNING_KEY_PASSWORD")
         }
 
-        getByName("debug") {
+        getByName(BuildType.DEBUG) {
             storeFile = rootProject.file("config/debug.keystore")
             storePassword = "oQ4mL1jY2uX7wD8q"
             keyAlias = "debug-key-alias"
             keyPassword = "oQ4mL1jY2uX7wD8q"
         }
     }
 
-    compileSdk = 32
+    compileSdk = Version.ANDROID_COMPILE_SDK_VERSION
     defaultConfig {
         applicationId = "co.nimblehq.ic.kmm.suv.android"
-        minSdk = 23
-        targetSdk = 32
-        versionCode = 1
-        versionName = "0.1.0"
+        minSdk = Version.ANDROID_MIN_SDK_VERSION
+        targetSdk = Version.ANDROID_TARGET_SDK_VERSION
+        versionCode = Version.ANDROID_VERSION_CODE
+        versionName = Version.ANDROID_VERSION_NAME
```

Comment:
Do you intend to fix this issue https://github.com/suho/kmm-ic/pull/80#discussion_r963301813 in this PR too? On both production and staging build workflows 🤔 

```
        with:
          gradlePath: android/build.gradle.kts
          versionCode: ${{ github.run_number }}
```

Reply:
Good catch! I updated it here: https://github.com/suho/kmm-ic/pull/83/commits/22945828d69ca242652197b0e5f03624fe60eefe

Not override the versionName :+1: